<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LDT Observing Log Autogenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="UTCDateTime">
      <property name="title">
       <string>Universal Time</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="txt_utcdate">
         <property name="font">
          <font>
           <family>Digital-7</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>UTC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="txt_utc">
         <property name="font">
          <font>
           <family>Digital-7</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LOCAL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="LocalDateTime">
      <property name="title">
       <string>Local Time (MST)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="txt_localdate">
         <property name="font">
          <font>
           <family>Digital-7</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>UTC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="txt_localtime">
         <property name="font">
          <font>
           <family>Digital-7</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LOCAL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QGroupBox" name="SunRiseSet">
      <property name="title">
       <string>Sun Status</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="txt_sunel">
         <property name="font">
          <font>
           <family>Digital-7</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>RISE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="txt_skystat">
         <property name="font">
          <font>
           <family>Digital-7</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SET</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Logging</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="dataLogTab">
          <attribute name="title">
           <string>Data Log</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="9" column="0" colspan="6">
            <widget class="QTableWidget" name="table_datalog">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="datalog_opendir">
             <property name="text">
              <string>Set Data Directory</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="txt_datalogdir">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>No Directory Chosen to Scan!</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QLabel" name="txt_datalogsavefile">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>No Output Filename!</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="datalog_savefile">
             <property name="text">
              <string>Set Log Output File</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="6">
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="5">
               <widget class="QPushButton" name="datalog_editkeywords">
                <property name="text">
                 <string>Edit Keywords ...</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QPushButton" name="datalog_cleartable">
                <property name="text">
                 <string>Clear Table</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="datalog_forcewrite">
                <property name="text">
                 <string>Force Write</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="datalog_addrow">
                <property name="text">
                 <string>Add a Blank Row</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="datalog_deleterow">
                <property name="text">
                 <string>Remove Highlighted Row</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="datalog_forceupdate">
                <property name="text">
                 <string>Force Update</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="4" rowspan="2" colspan="2">
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="1" column="3">
               <widget class="QComboBox" name="datalog_instrumentselect">
                <item>
                 <property name="text">
                  <string>LMI</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DeVeny</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>NIHTS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RC1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RC2</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="datalog_autoupdate">
                <property name="text">
                 <string>Autoupdate every:</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="txt_datalog_instrument">
                <property name="text">
                 <string>Instrument:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSpinBox" name="datalog_updateinterval">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="suffix">
                 <string> seconds</string>
                </property>
                <property name="value">
                 <number>60</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
